openedx/features/caliper_tracking/tests/current/edx.course.enrollment.upgrade.clicked.json

```diff
@@ -0,0 +1,21 @@
+{
+    "username":"honor",
+    "event_source":"browser",
+    "name":"edx.course.enrollment.upgrade.clicked",
+    "accept_language":"en-US,en;q=0.9,ur-PK;q=0.8,ur;q=0.7",
```

Comment:
This is also not present in the transformed code.

Reply:
Fixed